### PR TITLE
Show gateway locations retrieved from status messages in the map

### PR DESCRIPTION
**Pull request by kschiffer**
#### Summary
This PR adds functionality to display gateway locations from status messages in the gateway map in the Console.
It also streamlines how markers are displayed for end devices and gateways (both with a tooltip that gives information about the location source and trust)

image
(removed the exact lat/long for privacy reasons)

#### Changes

- Add store logic to paste status message locations into the gateway store
- Fix issue with prop types
- Use same utility function for generating markers from antenna locations

#### Testing

This can be tested as follows:

In the Console, navigate to a Gateway that uses GPS location in its status messages. On staging, this if for example:
https://tti.staging1.cloud.thethings.industries/console/gateways/kerlink-ibts-compact-1

The location in the overview should appear as soon as a status message is received (or already initially, if the event data retention included a status message)

#### Notes for Reviewers

As of now, all status updates are sent through insecure channels, which is why they cannot be trusted entirely. This is also why these locations are currently not used to update the gateway's location entry. I'm still in favor of at least displaying the location in the Console with a little warning and also to use them as a starting point when setting the location manually.

cc @adriansmares 

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [x] The steps/process to test this feature are clearly explained including testing for regressions.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/console/containers/packet-broker-networks-table/index.js
Comment (ryaplots) [inline pkg/webui/console/containers/packet-broker-networks-table/index.js:1]: Are these changes intentional? Seems like something from a previous PR.

File: pkg/webui/console/store/middleware/logics/gateways.js [hunk 1]
```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 import { createLogic } from 'redux-logic'
+import { eq } from 'lodash'
```
Comment (ryaplots) [inline pkg/webui/console/store/middleware/logics/gateways.js]: is this being used anywhere? I don't see it in the changes.

File: pkg/webui/console/store/reducers/gateways.js [hunk 2]
```diff
@@ -12,12 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { merge } from 'lodash'
+
```
Comment (ryaplots) [inline pkg/webui/console/store/reducers/gateways.js]: same comment here

**Review by ryaplots — APPROVED** [review]
Looks good! Just some small comments.

File: pkg/webui/console/containers/packet-broker-networks-table/index.js
Comment (kschiffer) [reply pkg/webui/console/containers/packet-broker-networks-table/index.js:1]: No, this is intentional because otherwise the location in the location form will update every time a new status message is received.

**Comment by KrishnaIyer** [comment]
> Manual testing.

Please add the steps to test this feature, including instructions for the setup.

**Comment by kschiffer** [comment]
Updated the description.

**Comment by KrishnaIyer** [comment]
Thanks!
